### PR TITLE
Updating readme with new CocoaPods instructions.

### PR DESCRIPTION
**Pull request by danbev**

### CODE REVIEW

**Comment by danbev** [comment]
@cvasilak @matzew Care to review this?

File: README.md [hunk 1]
```diff
@@ -42,4 +36,48 @@ is running. If it is not en error message simliar to this will be displayed:
...
+To use AeroGearSyncClient in your project add the following 'Podfile' to your project
+
+    source 'https://github.com/CocoaPods/Specs.git'
+
+    xcodeproj 'YourProjectName.xcodeproj'
+    platform :ios, '8.0'
+
+    pod 'AeroGearSyncClient', :git => "https://github.com/aerogear/aerogear-ios-sync-client.git", :branch => "master"
+
+    target 'AeroGearSyncClientTests' do
```
Comment (cvasilak) [inline README.md]: s/AeroGearSyncClientTests/YourProjectNameTests

**Comment by cvasilak** [comment]
apart from this small comment looks good :+1:

**Comment by danbev** [comment]
@cvasilak Thanks! I've update the PR with this change now.

**Pull request merged**